### DISCUSSION TITLE
I cant set proper time in waybar and keyboard layout

### DISCUSSION THREAD

**Discussion by Alejandrovsdev** [prasanthrangan/hyprdots#1535] [Q&A]
i already change the config file to add (es) keyboard layout but i can change it. And i already change the timezone and when i send date this show me the correct time but the waybar clock its not correct. anyone cant helpme with that?

**Comment by kRHYME7** [comment]
Have you tried changing the waybar layout?

**Reply by Alejandrovsdev** [reply]
yes, i tried it but its not working well, its only in waybar, in the terminal works fine.

**Reply by kRHYME7** [reply]
`   timedatectl set-local-rtc 1   `

it will give a warning, just ignore 

![image](https://github.com/prasanthrangan/hyprdots/assets/53417443/4fdd3bb2-4687-4a99-bd2b-10def8650900)

**Reply by rahim9500** [reply]
where can i change my keyboard layout in the files

**Reply by kRHYME7** [reply]
@rahim9500 

![image](https://github.com/prasanthrangan/hyprdots/assets/53417443/37c7d51d-ab17-47b5-968e-1defb6582e7d)

**Reply by Alejandrovsdev** [reply]
still not working only in waybar 
![Imagen de WhatsApp 2024-05-23 a las 12 36 34_d92a2d99](https://github.com/prasanthrangan/hyprdots/assets/112131333/9c6d951b-0fc9-4662-8ebf-84b28bd30ee3)
![Imagen de WhatsApp 2024-05-23 a las 12 38 08_aa0aff92](https://github.com/prasanthrangan/hyprdots/assets/112131333/20b5acd8-cd06-4ebf-8d5a-2f9c5f50fcfe)

**Reply by kRHYME7** [reply]
How about ` timedatectl set-timezone UTC `   ?  Remember you need to change waybar layouts for everything to be reloaded. 

In my case my waybar follows the ` date ` output so I'm a little bit confused too.

**Reply by Alejandrovsdev** [reply]
still not workin :C

**Reply by kRHYME7** [reply]
Plus in case if it is too stubborn, try to follow the waybar wik on setting the timezone  https://man.archlinux.org/man/extra/waybar/waybar-clock.5.en

the file that you're concerned with is this ` ~/.config/waybar/modules/clock.jsonc `

**Reply by rahim9500** [reply]
changed it in hyprland.conf and it is working. thx

**Reply by Alejandrovsdev** [reply]
i set te timezone via clock.jsonc and the waybar go to vacation to never return xd

**Reply by Alejandrovsdev** [reply]
i will try to delete all and reinstall arch linux with the dotfiles
